coral/src/app/features/connectors/request/schemas/connector-request-form.ts

```diff
@@ -0,0 +1,40 @@
+import attempt from "lodash/attempt";
+import isError from "lodash/isError";
+import z from "zod";
+
+const connectorRequestFormSchema = z.object({
+  environment: z
+    .string()
+    .min(1, { message: "Selection Error: Please select an environment." }),
```

Comment:
Ah, I like that. We should add that custom error to the topic request form, too 💡  (can be a different PR)

Reply:
I'd argue the "Selection Error" part is not needed. "Please select an environment" would propbably be enough.

Reply:
Yes, that sounds good!